src/core/math/line2/create.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Represents a unbounded line in 2D space, centered about an origin.
```

Comment:
I confess I do not understand this definition at all ?

Reply:
Line 2D and 3D have no end point, and continue forever in both positive and negative directions.

Reply:
ok but what does 'centered about an origin'  mean ? should it not be 'centered around an origin point' or centered around [0,0] ? etc

Reply:
Tried again... let me know if the description is better.